packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -129,13 +128,11 @@ class SystemRulesTable extends React.Component {
     }
 
     changePage = (page, itemsPerPage) => {
-        this.setState(() => (
-            {
-                currentRows: this.currentRows(page, itemsPerPage),
-                page,
-                itemsPerPage
-            }
-        ));
+        this.setState({
```

Comment:
I took the chance also to remove the callback in all of the `setState`. Before, we needed the callback because `currentRows` was making an async call to the Remediations API. But that's no longer relevant  since https://github.com/RedHatInsights/frontend-components/commit/3c6c2762227b96b80edf70b1aebcc39f13249683